src/pages/Stake/StakeV2.tsx

```diff
@@ -84,10 +84,28 @@ import useIsMetamaskMobile from "lib/wallets/useIsMetamaskMobile";
 import useWallet from "lib/wallets/useWallet";
 import "./StakeV2.css";
 import { GMX_DAO_LINKS, getGmxDAODelegateLink } from "./constants";
+import { UncheckedJsonRpcSigner } from "lib/rpc/UncheckedJsonRpcSigner";
 
 const { ZeroAddress } = ethers;
 
-function StakeModal(props) {
+function StakeModal(props: {
+  isVisible: boolean;
+  setIsVisible: (isVisible: boolean) => void;
+  chainId: number;
+  title: string;
+  maxAmount: bigint | undefined;
+  value: string;
+  setValue: (value: string) => void;
+  active: boolean;
+  account: `0x${string}` | undefined;
```

Comment:
I think we can use viem `Address` type instea of our own template string

Reply:
fixed